pkg/monitoring/metrics/virt-controller/vmstats_collector_test.go

```diff
@@ -507,4 +507,58 @@ var _ = Describe("VM Stats Collector", func() {
 			Expect(results[0].Labels).To(Equal([]string{"test-vm-nil-mode", "default", "test-vm-pvc-nil-mode", "null", "rootdisk"}))
 		})
 	})
+
+	Context("VM creation time metric collection", func() {
+		It("should collect VM creation time correctly", func() {
+			testTime := time.Now()
+
+			vm := &k6tv1.VirtualMachine{
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace:         "test-ns",
+					Name:              "test-vm",
+					CreationTimestamp: metav1.NewTime(testTime),
```

Comment:
there is no chance this can be null?

Reply:
so I don't think so since if vm created- even if not running there is a create timestamp, do you think maybe we should handle a case there is no timestamp? and set the metric value to 0?

Reply:
cc @machadovilaca

Reply:
```
// CreationTimestamp is a timestamp representing the server time when this object was
	// created. It is not guaranteed to be set in happens-before order across separate operations.
	// Clients may not set this value. It is represented in RFC3339 form and is in UTC.
	//
	// Populated by the system.
	// Read-only.
	// Null for lists.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
	// +optional
	CreationTimestamp Time `json:"creationTimestamp,omitempty" protobuf:"bytes,8,opt,name=creationTimestamp"`
```

I am not sure, just asking, can you review related docs?

Reply:
ok so it cannot be null but it can be 0 from what I understand since it's type metav1.Time which is struct. 
you think we should add 0 to the test?

I didn't add `if !vm.CreationTimestamp.IsZero()` before the append since  I think we should show 0 values in case there is a vm with 0 as create date. WDYT?

Reply:
Hi @avlitman, Please consider the use of this metric in Grafana where we want to show for each VM its create date or calculate the days from its creation.
If there is a 0 value you will need to filter it so that is will not show. 
If you dont report when the value is 0, it will simply not show and less data will be saved to Prometheus.
I don't see the added value in reporting it. Do you agree?

Reply:
If it's 0 then no need to add the metric, I agree. will fix the code

Reply:
fixed